### ISSUE TITLE
File Deleted Exception in CLI version

### ISSUE THREAD

**Issue by ldcouto** [overturetool/overture-fmu#30]
IF you run the export tool via the command line jar ( `java -jar fmu-import-export-0.1.1-SNAPSHOT-jar-with-dependencies.jar ...`, it will always throw an exception near the end of the export:

```java
Exception in thread "main" java.io.IOException: Unable to delete file: C:\Users\
CoutoLD\AppData\Local\Temp\overture-fmu679777694319453809\resources\modelDescrip
tion.xml
        at org.apache.commons.io.FileUtils.forceDelete(FileUtils.java:2279)
        at org.apache.commons.io.FileUtils.cleanDirectory(FileUtils.java:1653)
        at org.apache.commons.io.FileUtils.deleteDirectory(FileUtils.java:1535)
        at org.apache.commons.io.FileUtils.forceDelete(FileUtils.java:2270)
        at org.apache.commons.io.FileUtils.cleanDirectory(FileUtils.java:1653)
        at org.apache.commons.io.FileUtils.deleteDirectory(FileUtils.java:1535)
        at org.overturetool.fmi.Main$ConsoleProject.cleanUp(Main.java:368)
        at org.overturetool.fmi.Main.main(Main.java:189)
```java

**Comment by lausdahl** [comment]
I suppose that the output it generated was ok?

The error only happens on windows since it locks the files somehow. I'm aware of the error it reports but it should only be the final step of removing the temporary folder it used. It does not happen on *nix systems as far as I know.

**Comment by ldcouto** [comment]
You are right that it produces an FMU.

I wanted to set up a Jenkins job that generates the FMU and fails if
something goes wrong but with this exception being throw, it always fails.
If you can make the exporter produce some kind of success message before
the cleanup, I can grep for it and fail/pass based on that.

--luis

**Comment by lausdahl** [comment]
fixed in c3e3e10db8b8124cfdaeaec1c95a4f587bf7be1c
